src/routes/about/organization/[slug]/+page.server.ts

```diff
@@ -0,0 +1,134 @@
+import { error } from "@sveltejs/kit";
+import gql from "graphql-tag";
+import { print as printQuery } from "graphql";
+
+import contentfulFetch, { contentfulConnected } from "$lib/services/contentful";
+import documentWithParagraphData from "$lib/components/ContentfulRichText/__tests__/documents";
+
+import type { PageServerLoad } from "./$types";
+import type { OfficePage } from "$lib/services/contentful/schema";
+import type { OfficePageQuery } from "./$queries.generated";
+
+const query = gql`
+  query OfficePage {
+    officePageCollection {
+      items {
+        sys {
+          id
+        }
+        pageTitle
+        subheading
+        description {
+          json
+        }
+        servicesAndPrograms {
+          json
+        }
+        mailingAddress {
+          ... on ContentTypeLocation {
+            name
+            streetAddress1
+            streetAddress2
+            city
+            state
+            zip
+          }
+        }
+        contactsCollection {
+          items {
+            ... on Contact {
+              sys {
+                id
+              }
+              entityName
+              phone
+              email
+            }
+          }
+        }
+        metadata {
+          ... on PageMetadata {
+            slug
+            metaTitle
+            metaDescription
+          }
+        }
+      }
+    }
+  }
+`;
+
+export const load = (async ({ params }): Promise<OfficePage> => {
+  const { slug } = params;
+  if (contentfulConnected()) {
+    const data = await contentfulFetch<OfficePageQuery>(printQuery(query));
+    if (data) {
+      const officePages = data?.officePageCollection?.items as OfficePage[];
+      const matchedOfficePage = officePages.find(
+        (officePage) => officePage.metadata?.slug === slug
+      );
+      if (matchedOfficePage) {
+        return matchedOfficePage;
+      }
+    }
+    throw error(404);
+  } else {
+    return {
+      sys: { id: "0", environmentId: "", spaceId: "" },
+      contentfulMetadata: { tags: [] },
+      pageTitle: "Sample Office Page",
```

Comment:
Wasn't really sure where to put this test content since it doesn't match the current pattern like `Header` and `Nav` do (those have components, but the Office Page isn't a component). Open to suggestions here!

Reply:
I like this suggestion; turns out that SvelteKit will auto-ignore any underscore-prepended directories under `routes/` so I'll opt for the pattern we've been using elsewhere and move the sample response into `__tests__/OfficePageTestContent.ts`